### PR TITLE
Configurable warmup iterations for ectrans-benchmark

### PR DESCRIPTION
**Pull request by wdeconinck**
This PR should not be as difficult to review and includes:
- Add command line argument --niter-warmup to ectrans-benchmark
- Modify default to 3 as after this, experimentally the timings stay more or less constant
- Some cosmetics in printing and removal of spurious prints.

### CODE REVIEW

File: src/programs/ectrans-benchmark.F90 [hunk 1]
```diff
@@ -571,16 +569,16 @@ program ectrans_benchmark
 
 if (verbosity >= 0 .and. myproc == 1) then
   write(nout,'(" ")')
-  write(nout,'(a,i6,a,f9.2,a)') "ectrans_benchmark initialisation, on",nproc,&
+  write(nout,'(a,i0,a,f9.2,a)') "ectrans_benchmark initialisation, on",nproc,&
```
Comment (samhatfield) [inline src/programs/ectrans-benchmark.F90:572]: Haven't seen `i0` before. Does that automatically scale the width of the printed integer?

File: src/programs/ectrans-benchmark.F90 [hunk 2]
```diff
@@ -706,8 +704,8 @@ program ectrans_benchmark
     if (myproc == 1) then
       zmaxerr(:) = -999.0
       do ifld = 1, 1
-        write(nout,*) "znormsp", znormsp
-        flush(nout)
+        !write(nout,*) "znormsp", znormsp
+        !flush(nout)
```
Comment (samhatfield) [inline src/programs/ectrans-benchmark.F90:708]: I think we just delete these.

**Review by samhatfield — CHANGES_REQUESTED** [review]
Looks good!

File: src/programs/ectrans-benchmark.F90 (hunk 1 above)
Comment (marsdeno) [reply src/programs/ectrans-benchmark.F90:572]: Correct

File: src/programs/ectrans-benchmark.F90 (hunk 2 above)
Comment (wdeconinck) [reply src/programs/ectrans-benchmark.F90]: Done

**Review by marsdeno — APPROVED** [review]
All good for me

**Review by samhatfield — APPROVED** [review]

**Pull request merged**